release/kubernetes-manifests.yaml

```diff
@@ -450,9 +450,9 @@ spec:
       initContainers:
       - name: wait-frontend
         image: alpine:3.6
-        command: ['sh', '-c', 'set -x;  apk add --no-cache curl && 
-          until timeout -t 2 curl -f "http://${FRONTEND_ADDR}"; do 
-            echo "waiting for http://${FRONTEND_ADDR}"; 
+        command: ['sh', '-c', 'set -x;  apk add --no-cache curl &&
+          until timeout -t 2 curl -f "http://${FRONTEND_ADDR}"; do
+            echo "waiting for http://${FRONTEND_ADDR}";
```

Comment:
do we still need to update this?
(I assume you didn't mean to touch this file, as it is autogenerated.)

Reply:
I had accidentally updated the release/ manifests, then reverted that, I think that's just my linter.